revelio/settings_dev.py

```diff
@@ -163,8 +163,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/5.0/howto/static-files/
 
-# STATIC_URL = 'static/'
```

Comment:
remove this is not used

Reply:
I have uncommented this part. Will be used for Django Admin styling.